atable/src/components/ARow.vue

```diff
@@ -43,6 +44,20 @@ const tableData = inject<TableDataStore>(props.tableid)
 const rowEl = ref<HTMLTableRowElement>(null)
 const numberedRowWidth = tableData.numberedRowWidth.value
 
+const { x, y, style } = useDraggable(rowEl, {
```

Comment:
Without passing a `containerElement`, I think the composable is letting you drag the row outside the bounds of the table, which is causing some issues with the row data (random `undefined` rows)